### PR TITLE
testing/cmocka: forwarding cmocka log message to syslog channel

### PR DESCRIPTION
**Pull request by xiaoxiang781216**
## Summary

and ignore cmocka files in apps

## Impact

cmocka

## Testing

Pass CI

### CODE REVIEW

File: testing/cmocka/cmocka_platform.h [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.  See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ *
+ ****************************************************************************/
+
+#ifndef __TESTING_CMOCKA_CMOCKA_PLATFORM_H__
+#define __TESTING_CMOCKA_CMOCKA_PLATFORM_H__
```
Comment (pkarashchenko) [inline testing/cmocka/cmocka_platform.h:22]:
```suggestion
#ifndef __APPS_TESTING_CMOCKA_CMOCKA_PLATFORM_H
#define __APPS_TESTING_CMOCKA_CMOCKA_PLATFORM_H
```

File: testing/cmocka/cmocka_platform.h [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+#include <syslog.h>
+
+/****************************************************************************
+ * Pre-processor Definitions
+ ****************************************************************************/
+
+#define cmocka_vprint_message(f,a) vsyslog(LOG_INFO,f,a)
+#define cmocka_vprint_error(f,a) vsyslog(LOG_ERR,f,a)
+
+#endif /* __TESTING_CMOCKA_CMOCKA_PLATFORM_H__ */
```
Comment (pkarashchenko) [inline testing/cmocka/cmocka_platform.h:37]:
```suggestion
#endif /* __APPS_TESTING_CMOCKA_CMOCKA_PLATFORM_H */
```

Comment (xiaoxiang781216) [reply testing/cmocka/cmocka_platform.h]: Done.

File: testing/cmocka/cmocka_platform.h (hunk 1 above)
Comment (xiaoxiang781216) [reply testing/cmocka/cmocka_platform.h]: Done.

**Review by pkarashchenko — APPROVED** [review]

**Pull request merged**